### ISSUE TITLE
Loss is too high, and no result output by infer.py

### ISSUE THREAD

**Issue by PubMedKG** [suicao/coleridge-gpt#2]
![image](https://github.com/suicao/coleridge-gpt/assets/56544472/735f07bd-9407-406e-a5ea-079831b80413)
![image](https://github.com/suicao/coleridge-gpt/assets/56544472/93d32931-ef7f-451d-8748-30f79ad589ba)

Hi suicao,

I want cite your code for dataset NER, but I don't know why loss is too high, and no result output by infer.py.

I have tried the code for 5 times at least.  Could you please check the code?

Thank you very much.

**Comment by suicao** [comment]
Sorry for the late reply, have you fixed the issue?
It seems to me the `p_mask ` parameter in the forward function of the model messed with the cross entropy loss, you can send `p_mask = None` from the training loop and the loss would converge.

**Comment by PubMedKG** [comment]
> Sorry for the late reply, have you fixed the issue? It seems to me the `p_mask ` parameter in the forward function of the model messed with the cross entropy loss, you can send `p_mask = None` from the training loop and the loss would converge.

Oh, thank you, it works for me. You are really  a cool man!
